### PR TITLE
created checkbox, radio button, icon button and fixed

### PR DESCRIPTION
**Pull request by AziatAbd**

### CODE REVIEW

**Comment by AziatAbd** [comment]
Approve

File: src/components/UI/Checkbox.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import {
+   FormControlLabel,
+   FormGroup,
+   Checkbox as MuiCheckbox,
+   styled,
+} from '@mui/material'
+
+const Checkbox = ({ label = '', checked = false, changeChecked }) => {
+   const changeHandler = (e) => {
```
Comment (alisher-zhuman) [inline src/components/UI/Checkbox.jsx]: в одну строку по браткски

File: src/components/UI/IconButton.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { IconButton as MuiIconButton } from '@mui/material'
+
+const IconButton = ({ children, ...rest }) => {
```
Comment (alisher-zhuman) [inline src/components/UI/IconButton.jsx]: delete return and {}

File: src/components/UI/RadioButton.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { FormControlLabel, Radio, RadioGroup, styled } from '@mui/material'
+
+const RadioButton = ({ options, value, onChange }) => {
```
Comment (alisher-zhuman) [inline src/components/UI/RadioButton.jsx]: toje samoe udali ?

File: src/components/UI/RadioButton.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+   return (
+      <RadioGroup value={value} onChange={onChange}>
+         {options.map((option) => (
```
Comment (alisher-zhuman) [inline src/components/UI/RadioButton.jsx]: destructuring

**Pull request merged**